### DISCUSSION TITLE
neuralamp LV2 crashes Reaper on Ubuntu Studio 22.04

### DISCUSSION THREAD

**Discussion by guidokon** [mikeoliphant/neural-amp-modeler-lv2#53] [Q&A] [answered]
Hello ,

i tried to get the neuralamp lv2 to run in Reaper with a fresh install and compile from github on 5.Jan.2024. 
 
After scan the lv2 directory and  then try to add the neuralamp lv2 on a track reaper crashes.

from dmesg i read 

37.039479] process '/opt/REAPER/reaper' started with executable stack
[  289.356250] traps: reaper[3000] trap invalid opcode ip:7f7df0c0488b sp:7fff0e0c7ba0 error:0 in neural_amp_modeler.so[7f7df0bfb000+37000]

Ububtu Studio Version

uname -v
#18~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Thu Nov 16 11:27:56 UTC 2

Reaper: version: v7.07

I tried the add to reaper from /usr/lib/lv2 and later after removing it from /usr/lib/lv2 from $HOME/.lv2 and it crashes both times.

In $HOME/.lv2 i changed the owner of all entries to my userid.

I am not sure if i have to  set something for  rdfs:range atom:Path in  neural_amp_modeler.ttl. 
Currently its set as i got it from github :   rdfs:range atom:Path.

Can you please give some help on it ?

Kind regards
Guido

**Comment by mikeoliphant** [comment, marked as answer]
Do you have an older x86 CPU? If so, see this issue: https://github.com/mikeoliphant/neural-amp-modeler-lv2/issues/52

**Reply by guidokon** [reply]
Hello Mike,
thanks you very much. The crashing is gone.  Great work !
Kind regards
Guido

**Comment by mikeoliphant** [comment]
👍
